### PR TITLE
chore: add GCR build script and remove workflow

### PR DESCRIPTION
**Pull request by deepak-deepsource**
* Remove GH action based push to container reg

### CODE REVIEW

File: .deepsource.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version = 1
+
+[[analyzers]]
+name = "swift"
```
Comment (deepak-deepsource) [inline .deepsource.toml:4]:
```suggestion
```

File: cloudbuild_depl.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      - -t
+      - us.gcr.io/deepsource-production/swift-format:$TAG_NAME
+      - -t
+      - us.gcr.io/deepsource-production/swift-format:latest
+      - .
+
+images: ['us.gcr.io/deepsource-production/swift-format:$TAG_NAME', 'us.gcr.io/deepsource-production/swift-format:latest']
+
+options:
+  machineType: 'E2_HIGHCPU_32'
```
Comment (srijan-deepsource) [inline cloudbuild_depl.yaml:17]: use 8 cores.

File: cloudbuild_depl_dev.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  - name: "gcr.io/cloud-builders/docker"
+    args:
+      - build
+      - --tag=us.gcr.io/deepsource-dev/swift-format:dev
+      - .
+
+images: ['us.gcr.io/deepsource-dev/swift-format:dev']
+
+options:
+  machineType: 'E2_HIGHCPU_32'
```
Comment (srijan-deepsource) [inline cloudbuild_depl_dev.yaml:13]: same.

File: cloudbuild_depl.yaml (hunk 2 above)
Comment (deepak-deepsource) [inline cloudbuild_depl.yaml:17]:
```suggestion
  machineType: 'E2_HIGHCPU_8'
```

File: cloudbuild_depl_dev.yaml (hunk 3 above)
Comment (deepak-deepsource) [inline cloudbuild_depl_dev.yaml:13]:
```suggestion
  machineType: 'E2_HIGHCPU_8'
```

**Review by srijan-deepsource — APPROVED** [review]

**Pull request merged**